ramp_planner_new/PathPoints pathPoints
visualization_msgs/Marker[] markers
uint32 pathId